### ISSUE TITLE
Dependency on `FSharp.Core 4.3.1`

### ISSUE THREAD

**Issue by WonkiDonk** [heartysoft/restsharp.portable.hal#6]
The nuget packages have a dependency on `FSharp.Core 4.3.1`, however, NuGet reports *4.2.1* is the latest version of `FSharp.Core`. As a result, I can't install the NuGet package as 4.3.1 cannot be resolved.

**Comment by isaacabraham** [comment]
I suspect this is an misnomer. The F# assembly with a version of "4.3.1.0" is actually F# 3.1 (the first number can be ignored!). This maps to the FSharp.Core nuget package of 3.1.2.5.

**Comment by ashic** [comment]
Thanks @isaacabraham . I suspect you're right. When this was written, F# was at version 3. Don't you just love .NET versioning?

**Comment by forki** [comment]
yes this major source of confusion. In this package (and in others as well) dll version doesn't match package version
